widgets.html

```diff
@@ -147,6 +153,35 @@ <h3>Installation</h3>
           With pip:
           {% highlight bash %}pip install nglview
 jupyter nbextension enable --py --sys-prefix nglview{% endhighlight %}
+        </div>
+        <div class="tab-pane" id="k3d">
+          <div class="jupyter-widget-header">
+            <span class="gallery-title">K3D-Jupyter</span>
+          </div>
+          <p>
+            <a href="https://github.com/K3D-tools/K3D-jupyter">K3D</a> is a Jupyter widget to 3-D plotting using WebGL.
+          </p>
+          <h3>Example</h3>
+          {% highlight python %}{% include_relative assets/k3d-example.py %}{% endhighlight %}
+          <script type="application/vnd.jupyter.widget-state+json">
+              {% include_relative assets/k3d-example.json %}
+          </script>
+          <script type="application/vnd.jupyter.widget-view+json">
+          {
+              "model_id": "d3a24bac28644484b2aad2bdf34f31bf"
+          }
+          </script>
+          <script type="application/javascript">
+              function adjustK3D() {
+                  setTimeout(function() {
+                      window.dispatchEvent && window.dispatchEvent(new Event('resize'));
+                  });
+              }
```

Comment:
It seems a tiny bit strange that we have to send a resize event to the window to display the widget correctly? Why do we have to do that?

Reply:
I agree. I follow the pattern from BeakerXTable - adjustBeakerXTable(). 
The widget automatically fit to the container size - thats good because that make a widget a full RWD handling. Normally everything works fine in jupyter env. In widget gallery widget is initialize inside hidden element (css display:none) and targetDOMNode.offsetWidth is 0 in that case.

I prepared jsfiddle to illustrated this: https://jsfiddle.net/s1gcpzv8/9/

Reply:
That makes sense, thanks for the explanation.